packages/dev-env/package.json

```diff
@@ -25,13 +25,10 @@
     "express": "^4.18.1",
     "get-port": "^6.1.2",
     "level": "^8.0.0",
-    "sqlite3": "^5.0.11",
-    "ucans": "^0.9.1",
-    "uint8arrays": "^3.0.0"
+    "sqlite3": "^5.0.11"
   },
   "devDependencies": {
     "@types/express": "^4.17.13",
-    "@types/inquirer": "^8.2.1",
```

Comment:
I saw you removed `inquirer` so I also removed the types. That good?

Reply:
Yep!